### PR TITLE
prefer class methods over instance methods for the pacemaker model

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

**Comment by MaximilianMeister** [comment]
Included the important bit from https://github.com/crowbar/crowbar-ha/pull/141 here now

**Comment by MaximilianMeister** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;8X7elkka&#x2F;82-cleanup-upgrade-related-mess"> Cleanup upgrade related mess</a>

File: crowbar_framework/app/models/api/pacemaker.rb [hunk 1]
```diff
@@ -16,44 +16,50 @@
...
+            failed_actions[n.name] = ssh_retval[:stdout]
+          end
         end
+        ret["crm_failures"] = crm_failures unless crm_failures.empty?
+        ret["failed_actions"] = failed_actions unless failed_actions.empty?
+        ret
+      end
+
+      def repocheck
+        Api::Node.new.repocheck(addon: "ha")
```
Comment (jsuchome) [inline crowbar_framework/app/models/api/pacemaker.rb]: I assume you're waiting for merge of that PR with the changes in `Api::Node`...

Comment (MaximilianMeister) [inline crowbar_framework/app/models/api/pacemaker.rb]: actually not, thanks. fixed

**Comment by MaximilianMeister** [comment]
rebased

**Pull request merged**